ras/ras_ppcutils.py

```diff
@@ -428,6 +433,35 @@ def test_lparstat(self):
                 self.log.info("%s command passed" % cmd)
                 self.fail("lparstat: Expected failure, %s command exeucted \
                           successfully." % cmd)
+        output = process.system_output("lparstat -E 1 1").decode("utf-8")
+        input_string = ''
+        for line in output.splitlines():
+            if 'GHz' in line:
+                input_string = line
+                # Define the regular expression pattern
+                pattern = (r'(\d+\.\d+)\s+(\d+\.\d+)\s+\d+\.\d+GHz\[\s*(\d+)%\]\s+'
+                           r'(\d+\.\d+)\s+(\d+\.\d+)')
+                # Find all matches in the input string
+                matches = re.findall(pattern, input_string)
+                for data in matches:
+                    actual_busy = float(data[0])
+                    actual_idle = float(data[1])
+                    normal_idle = float(data[3])
+                    normal_busy = float(data[4])
+                    normal = normal_idle + normal_busy
+                    freq_percentile = float(data[2])
+        if (actual_busy > 0) and (actual_idle < 100):
+            self.log.info(
+                "Actual busy and idle values are displaying  correctly")
+        else:
+            self.fail("busy value and idle values are not \
+                       displaying correctly")
+        if normal == freq_percentile:
+            self.log.info("normal busy and idle are matching with the \
+                           percentage under frequency")
+        else:
+            self.fail("normal busy and value percentage are not matching \
```

Comment:
Normalised busy plus idle value does not match with Frequency (percentage)

Reply:
@sacsant As Suggested above changed the sentences and re-pushed it again